.github/workflows/test.yml

```diff
@@ -0,0 +1,70 @@
+name: IBC Testing with Local Chains
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  ibc-testing:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build Docker image
+        run: docker build -t a3p:local .
```

Comment:
why are we using a local a3p image instead of the latest one deployed?

Reply:
Are you asking why we aren’t using the `a3p:latest`? I’m not sure, I’ve been following the instructions in the README. But, I noticed that we are using `a3p:main` during our tests.

@toliaqat, is there a specific reason we're not opting for `a3p:latest`?